src/server/distributed_engine.rs

```diff
@@ -6,6 +6,7 @@ use crate::common::{
 };
 
 use crate::rpc::client::Client;
+use libc::off_t;
```

Comment:
Do not use libc types on server. The type converting should only occur on client.

Reply:
done